lib/web/oidcidp_test.go

```diff
@@ -69,3 +72,29 @@ func TestOIDCIdPPublicEndpoints(t *testing.T) {
 	require.Equal(t, key.Alg, "RS256")
 	require.NotNil(t, key.KeyID) // AWS requires this to be present (even if empty string).
 }
+
+func TestThumbprint(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	// Proxy starts with self-signed certificates.
+	lib.SetInsecureDevMode(true)
```

Comment:
Is this safe to do in a parallel test? I would think it's not.

Reply:
🙏 Removed.

Reply:
it still looks parallel?